### ISSUE TITLE
Special characters

### ISSUE THREAD

**Issue by Mingo3347** [LNReader/lnreader-plugins#878]
### Source information

فضاء الروايات (riwayat)

### Steps to reproduce

Open any novel in (فضاء الروايات) source and the chapters contains tons of special characters between every single paragraph making it very annoying........this didn't exist in the previous version of the app but only after I updated to the beta version of 2.0.0
![Screenshot_20240407-043850_LNReader](https://github.com/LNReader/lnreader-plugins/assets/108782331/f21ccc91-0b96-4e3b-9d39-b5cee0a99f0d)

### Expected behavior

Only the chapters and the content of the novel should be visible

### Actual behavior

Special characters and warnings from the source 

### LNReader version

2.0.0

### Android version

Android 11

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to version **[1.1.19](https://github.com/LNReader/lnreader/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] If this is an issue with the app itself, I should be opening an issue in the [app repository](https://github.com/LNReader/lnreader/issues/new/choose).
- [X] I will fill out all of the requested information in this form.
